modules/parsers/nasm/nasm-parser-struct.h

```diff
@@ -36,6 +36,7 @@ typedef union {
     uintptr_t arch_data;
     struct {
         char *contents;
+        yasm_utfenc enc;
```

Comment:
should this be in the `str` struct or its own field in the union, as it is only set when reading the `__utf*__` keywords, and _not_ when reading the actual string literal?